dist/index.html

```diff
@@ -1,19 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
-    <title></title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <title>To-Do list</title>
 <script defer src="runtime.js"></script><script defer src="bundle.js"></script></head>
+
 <body>
-    <main>
-        <p id="header">Today's To Do <i class="bi bi-arrow-repeat"></i></p>
-        <input type="text" id="input-task" placeholder="Add to your list...">
-        <ul id="todos-list">
-        </ul>
-        <p id="completed">Clear all completed</p>
-    </main>
+  <main>
+    <h2 id="header">Today's To Do <button id="refresh-list"><i class="fa-solid fa-arrows-rotate"></i></button></h2>
+    <input type="text" id="new-todo" placeholder="Add to your list...">
+
+    <ul id="todo-list">
+    </ul>
+
+    <h3 id="clear-completed">Clear all completed</h3>
```

Comment:
- [ ] It would be better if we implement the `clear all completed` functionality as in the project requirements beloq
![Screenshot from 2023-05-11 13-56-10](https://github.com/Zohra-Neda/To-Do-List/assets/46388113/f5ade0f0-1d6d-4cd7-81bd-be7093286df1)